src/store/index.js

```diff
@@ -1,2 +1,29 @@
+/*
+ * Use store pattern instead of Vuex since this is a plugin
+ * and instantiated externally
+ **/
+
 import Vue from 'vue'
-export default Vue.observable({editMessage: null})
+
+const store = {
+  state: Vue.observable({
+    editMessage: null
+  }),
```

Comment:
This can be accessed directly in components, or its keys can be mapped to computed props using the `mapState` helper function below. This is analogous to Vuex.